### PR TITLE
Fix SQL error for duplicate rows in [tools] detect_duplicated_commentids.php

### PR DESCRIPTION
**Pull request by GeorgeMurad**
## Update the select query from pselectRow tp pselect on line 105 under [tools] detect_duplicated_commentids.php file

- [x] Have you updated related documentation?yes, [tools] detect_duplicated_commentids.php file

#### Issue: https://github.com/aces/Loris/issues/8060

### CODE REVIEW

**Comment by ridz1208** [comment]
@GeorgeMurad The array you are getting now will be an array of arrays as opposed to the pselect row call that simply returns a 1D array. you need to handle the code below properly

File: tools/detect_duplicated_commentids.php [hunk 1]
```diff
@@ -102,7 +102,7 @@
             $candid = $candidate['CandID'];
             $pscid  = $candidate['PSCID'];
             foreach ($cohortids as $cohortid) {
-                $session_info = $DB->pselectRow(
+                $session_info = $DB->pselect(
```
Comment (kongtiaowang) [inline tools/detect_duplicated_commentids.php:105]: $session_info = $DB->pselectRow(
                    "SELECT DISTINCT s.Visit_label,s.ID from session s
                    JOIN candidate c on (c.candid=s.candid)
                    JOIN flag f on (f.sessionid=s.id)
                    WHERE s.candID = :cid AND f.test_name = :fname AND
                    s.cohortid = :subid 
                    limit =1",
                    [
                        'cid'   => $candid,
                        'fname' => $instrument,
                        'subid' => $cohortid['cohortid'],
                    ]
                );

**Comment by ridz1208** [comment]
@driusan this should probably go to 24.1, please confirm

**Comment by driusan** [comment]
Yes, I think it should go to 24.1

**Comment by driusan** [comment]
@GeorgeMurad can you please rebase this as requested

**Comment by ridz1208** [comment]
merged in 24.1

**Pull request closed without merging**